.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu, macos, windows]
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

Comment:
We should probably keep 3.8 support for netlify.

Reply:
This is becoming quite a nuisance. Can we use a venv to build?

```
brew install python3
PYTHON=$(brew --prefix python)/libexec/bin/python
$PYTHON -m venv .venv
source .venv/bin/activate
```

or similar?

Reply:
It hasn't a problem for me yet. If it does, I will look into it. For now, I will just revert it.